lib/OpenQA/CacheService.pm

```diff
@@ -105,8 +128,10 @@ sub run {
     my $app = __PACKAGE__->new;
     $ENV{MOJO_INACTIVITY_TIMEOUT} //= 300;
     $app->log->debug("Starting cache service: $0 @args");
+    $app->{_main_pid} = $$;
```

Comment:
There are cleaner storage options.
```perl
$app->config->{manager_pid} = $$;
```
or
```perl
$app->defaults->{manager_pid} = $$;
```

Reply:
This seems a little bit weird. This value is not a configuration and not a default. But I can use one of them for the sake of doing it in "the framework's way". However, I really prefer to keep the name "main_pid". "manager_pid" is misleading because in case the service is not started in prefork mode there is no server manager.

Reply:
`$app->defaults` is the data the stash is initialized with for every request, so basically an app level scope for arbitrary data. That's probably the most appropriate place. I don't care at all about the name, but since we are bikeshedding anyway, how about `service_pid`? 😁